### PR TITLE
upgraded athenaeum and linted to fix error

### PR DESCRIPTION
**Pull request by magcurt8**
Upgrading athenaeum dependencies to resolve security issues

[TECHDEBT-228](https://policygenius.atlassian.net/browse/TECHDEBT-228)

### CODE REVIEW

File: src/atoms/Tooltip/__tests__/tooltip.spec.js [hunk 1]
```diff
@@ -34,8 +34,9 @@ describe('Tooltip', () => {
     describe('when headerText is provided', () => {
       it('renders the provided headerText prop', () => {
         const headerText = 'Fear is the mind-killer';
-        props = { headerText: headerText };
-        component = shallow(<Tooltip {...props}/>);
+
+        props = { headerText };
```
Comment (jmcolella) [inline src/atoms/Tooltip/__tests__/tooltip.spec.js]: nit: can you remove this change to the spacing? unless it breaks the linter

File: package.json [hunk 2]
```diff
@@ -40,7 +40,6 @@
     "identity-obj-proxy": "^3.0.0",
     "lazysizes": "^4.0.1",
     "lodash": "^4.17.4",
-    "npm": "^6.4.1",
```
Comment (gummoe) [inline package.json]: Whoa - so we were able to remove it!?

Comment (magcurt8) [reply package.json]: YESSIR!

File: src/utils/fieldUtils/renderTooltip.js [hunk 3]
```diff
@@ -10,12 +10,12 @@ export default function renderTooltip(tooltip, className, iconClassName) {
...
         ...overrides
       }),
-      icon: (
-        <Icon
-          icon='tooltip'
-          className={iconClassName}
-        />
-      )
+      icon:
+  <Icon
```
[the remark this answers is not in the document]
Comment (magcurt8) [reply src/utils/fieldUtils/renderTooltip.js]: yeah these two changes are affecting the linter :(

**Pull request merged**